docs/reference/setup/install/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "2.2"
 services:
   es01:
     container_name: es01
-    image: { docker-image }
```

Comment:
I don't understand why we're changing this (aside from fixing the whitespace).

It looks like `{docker-image}` does the same thing:

https://github.com/elastic/elasticsearch/blame/master/docs/Versions.asciidoc#L12

Reply:
It wasn't rendering correctly in the output: https://www.elastic.co/guide/en/elasticsearch/reference/8.0/docker.html#docker-docker-compose

![image](https://user-images.githubusercontent.com/25848033/142687709-288fee50-e103-43c7-8434-14b9576833fa.png)

Reply:
I think that's just due to whitespace. It should be `{docker-image}` instead of `{ docker-image }` (no inner whitespace).

`{docker-image}` renders correctly when I tested it locally:

<img width="697" alt="Screen Shot 2021-11-19 at 3 43 39 PM" src="https://user-images.githubusercontent.com/40268737/142689326-e2997051-e75d-485f-81b2-d348ad1270e3.PNG">